content/information-distribution/article.md

```diff
@@ -5,12 +5,10 @@ weight: 3
 
 An article is a news item published by a
 [source](/information-distribution/sources) to the Matrix room. It **must**
-include the item's content, title, author, date (formatted following the
-[ISO8601](https://tools.ietf.org/html/rfc3339) standard) and original URL (or an
-empty string if the item was only published through Informo), along with a
-signature coming from its source, and be published on the publication name space
-that has been declared by its source. The article **can** also include optional
-properties, such as the item's description, read duration, NSFW marker, etc..
+include the item's content, title, author, date (in milliseconds since epoch)
+and original URL (or an empty string if the item was only published through
+Informo). The article **can** also include optional properties, such as the
+item's description, read duration, NSFW marker, etc..
```

Comment:
Only `title` and `content` must be set, other fields are optional. See table below

Reply:
Indeed. Fixed in 648c5539a1437dd5908ced88a302ab9009a02dbe.